docs/transpile/customizing-transpilation.ipynb

```diff
@@ -0,0 +1,468 @@
mie0zJo6IiKhUWrduDQCYN28ejEYjACA5ORn79+9HamoqatasCQA2saVLl2LIkCFo2LAhpk6dajeuVqudsEXuYcCAAaw7IiIiompWUtvXm9pngiRJkrMLQURE7mfq1KlYtmwZNmzYYDMxoKNYaeIkz13qTq8FDiy2XtZqMKDycU55iIiIqPp4YjugIm1fd8e3qhERUZl99dVXWLhwIbZs2WJzc3QUK02c5LHuiIiIiKpfRdq+nsCtRhxlZGQ4uwhERAQgLi4ORUVFVs9uL1u2DPHx8Q5jJa1LjrlT3RmLBJzbXMNqWUzXmxA1btPsICIionLytHZARdq+ZVGrVq1KKW9lY+KIiIiIKp2nNRiJiIio9NgOKB9XTRzxUTUiIrKiyNVCvJlvs9zkq4Yx1Beq9BzblQQB+vo1oLx4C4LJZBM2RAWjIN8HBi3fyVBWClGCf4QOqgu3ANg2tvTRISjI0cCoc626NekFGHWC1bK8DDUUKtdpMCqUEvwj9M4uRrXL0OtwQqu1WR4simjh5++EElF1O1CQj9t3JnO9V2MfX0SqVE4oEVWnPKMR+wts7/OiIKBTQKATSkTV7aS2EFf1tve/Omo1GmrceBKichKv5UFRUGSz3BjqBwiCfLs40nvOFyaOiIjIiibtKvx2nrdZro8KQkGnBxC09A+76914/VEELT4EwU4sd2A8rp6IQMF1z3rDRHUQRAkP9buMoKWH7cazn2mNK/siUJTt+p29S7+GOrsIVpQ+RjQedN3Zxah2K7Nv4eOMdJvlcRofbIxt4oQSUXV789IFnNfZdpI+iKqDkeE1nVAiqk6HCwvw7PkzdmOnm8VDrXCtHyKo8n1xLQOrc7Jtlg8JrYGpdepXf4GczH/7KaguZtssL2xTF5JKlG0X5w5rXfWFcxG8KhARkRVBa7C7XKHVQ1FkPwYAittau0kjABCKDDajT6h0JKMAqcBBvWsNLjfayF14a73l2Blp4mg5eZ4co/1rCo8B75Ats/8BINfEY8AbZPM+YEUotH9OCEUGCDJtX4XWu0Yse2eLiYiIZAl6+40GQW+EoHOQwMjXyX+nzgiTgbecciuUb8gJOgNMeiblykMyCZBsn6z0eHl2HicFgAKZ5eR58mX2dT6TBl7B0bkud2yQZymQSRB56/6XbfvqjBB08jFvwlY8ERFZkbt5Qm8y/8mt52A0kmAwwmRgcqO8pCIH8wIZTKzbCvDGutPKdAwKvTGL5oWMkgSdzLtxtDwGvEKhg+SA3PWBPIvc9d7RseHJBINMcshgkm8XG7yrrpg4IiIiK4LMjVAwmGRvrAAcjkaC3giTgzCVoMhB40RvAiTvS35UFpPR++pOrmOolyQY3Odlu1ROjhID3tpp9DY8BqjQJJM89tb9L/fDqMHooF3MEUdEROTN5H51MZpkb56A4xFHMJggmbyvg15pHIz0knTs6FeE5I2JIwejSoq8tdPgRRzvf15PvIHWQYK4iMljr1Akcx2QW+7pBKP8j6aQixklwIuumXyrGhERWZG7eQIA7hmuu/bMQQxe+X/QF8+JMdX6oyv6T7CO30cpqmAw2p9YcONn3nMjLhW9dX1Y1f1U24+z/krPG6d0cZQc0Eom+EOsxtJQdStp/5Pnc5Qg9toRJ15G7jqg9aJEiIUkORxtD+HuD0wltX1VKhX0evtt26tXr1ZKcZ2l3COOVq1ahQ4dOsDPzw9hYWEYOHAgTp48ieTkZAiCgJSUlEosJhERVRuDCTcKbyPhh3fwya4VViFBb7TEJu9cJpsUAoD15w45jMsljciO+/ZJSXVLpedtI47Gjh2LjV2egHT5sk3M+NMiPFS3HjZu3OiEklF1GDZsGLo8/DCkW7dsYlJ+Ptb37otevXrBxOSBxxo7diymt+sgew14OjaO1wAPNmzYMDRu3BjamzdsYlJ+Pi4MHOJ91wCjOVlmt+1rNI84Km3bVy5p5AnKlTiaM2cOkpKSsHv3brRt2xbdu3fH/v37kZCQgDNnzlR2GYmIqBoJRgmLj+/CwLh2eCqmBZ75eQYCP38OJ29ehaA3WWKtaz1gtd5b/Z/D5EefxsR2/ZD+6hz8eOw3q/gz3d7FkuTrWJJ8HbF1WsvGliRfr/JtdAenrxxEoS7f/B/31PvE9v2w6cKfVp+9t/6e6fovdJ8gYNfR1U4otfvxtsTRpEmToNBoYJw+y2q5dDUDpgUL8XjPnujevbuTSkdVbcqUKdDr9TDO+cYmZpqbAn12Nj777DMoFJzNwlNNmjQJSh8f2WtAqyef5DXAg02ZMgU6nQ55X35tEzPNTYEpJ8frrgHFI+3ttX1PZ16CYLjbBqsXFG617vixbyAtLQ1paWkYOnSodWz8eEssLS2t2ranqpT5iLhw4QLGjx8PlUqFdevWYfv27fjpp59w6tQpJCYmYv78+VVRTiIiqi5GE346/hserdMYvZZMxrmca/BVqvHaxv9CMJhjQ5s8gqt51r9YT10xDz+k7cDz8U/g+I105BQVAADUovmp6IWbP8bRCztRUHQbZ9IPAwBqBtexigX7hyPY3/qm7KosSZ0qkHnrAt7+pjsm/qcHCovyAINkqXcA6Fg7DgCgsVO3S3ZMQ+eHB6J908QqK19Vqsp6tcfbnswJDw9H9MsvQjr8B0wbNlmWG2fOBkQRY99714mlo6pWr149PPPG65C2bodp3wHLctOJkzCtWYeovwxBs2bNnFhCqmrh4eFo+dpY2WvAwIn/cGLpqKrVq1cPf/3rX2Hcus3uNUA9cID3XQPuJI7stX1fXzHHql286vR+q1WnfzUL+/fvR1hYmGW0kY+Pjzk2fbolFhYWVr3bVAXKnDiaO3cutFothg4dih49eliWq1QqTJ8+HX5+fjbrpKSkQBAEm79t27ZZPvP999+jTZs2CAkJgb+/P1q1aoWffvqpfFtFRETldu56OlQKJabvX4cbhXkI0fjjo85/wbZLR/HFmoVQKZSoFxSO9ef/AACEavwBACbJhHM
```

Comment:
I can't tell if this is just an issue with the notebook review display, but the "Next steps" section should be inside an admonition like it is on the rest of the pages